builtin/providers/aws/structure.go

```diff
@@ -247,6 +248,20 @@ func flattenHealthCheck(check *elb.HealthCheck) []map[string]interface{} {
 	return result
 }
 
+// Flattens a stream specification into something that flatmap.Flatten()
+// can handle
+func flattenStreamSpecification(spec *dynamodb.StreamSpecification) []map[string]interface{} {
```

Comment:
I think it's best to name this ``flattenDynamoDBStreamSpecification`` so it's clearer by the name what AWS product this relates to. 

It seems like the convention within the AWS provider is that EC2 and VPC resources don't have prefixes but everything else is prefixed with the service, so you can see ``flattenSecurityGroups`` below (an EC2/VPC concept) but later on ``flattenElastiCacheParameters`` and ``flattenEcsLoadBalancers``. (I'd concede that ``flattenResourceRecords, which is for Route53, is an exception to this general rule... so feel free to ignore me if you want. :) )